manifests/examples/linux-bridge.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkState
+metadata:
+  name: node01
+spec:
+  managed: true
+  nodeName: node01
+  desiredState:
+    interfaces:
+      - name: eth1
+        type: ethernet
+        state: up
+      - name: br1
+        type: linux-bridge
+        state: up
+        bridge:
+          options:
+            group-forward-mask: 0
+            mac-ageing-time: 300
+            multicast-snooping: true
+            stp:
+              enabled: true
+              forward-delay: 15
+              hello-time: 2
+              max-age: 20
+              priority: 32768
+          port:
+            - name: eth1
+              stp-hairpin-mode: false
+              stp-path-cost: 100
+              stp-priority: 32
+status:
```

Comment:
actually, we don't need status section while applying at all. we should drop it from examples

Reply:
done

Reply:
it's still here

Reply:
This is the old file, new one is create-br1-linux-bridge.yaml